spring-security/src/test/java/com/sap/cloud/security/spring/config/OAuth2ServiceConfigurationPropertiesTest.java

```diff
@@ -58,8 +61,38 @@ void setGetCertificateAndKey() {
 		assertEquals(ANY_VALUE, cutIas.getProperty(CERTIFICATE));
 	}
 
+	@Test
+	void setGetCertificateAndKeyXSUAA() {
```

Comment:
whats the difference between setGetCertificateAndKeyIAS and setGetCertificateAndKeyXsuaa?
furthermore, i would expect cut.getClientSecret() is SET

Reply:
Internally the only difference is the service field IAS or Xsuaa, but I think to represent both service differences, I would actually change it slightly: for setGetCertificateAndKeyIAS I wouldnt set the clientsecret, only for setGetCertificateAndKeyXSUAA, but then again it doesn't seem to be the right place to test it. I'll combine the both then.